server.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+
+const db = require('./db');
+
+const server = express();
+
+server.use(express.json());
+
+// Sanity check endpoint
+server.get('/', (req, res) => {
+  res.status(200).json({ api: 'up' });
+});
+
+server.get('/games', (req, res) => {
+  try {
+    res.json(db.data);
+  } catch (err) {
+    res.status(500).json({ Message: 'Looks like something went wrong' });
+  }
+});
+
```

Comment:
You're meeting all the spec's requirements Jesse 👍 Code is clean and easy to parse on my end 🔥